css/style.css

```diff
@@ -1,19 +1,465 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+    --color-main: #E16337;
+    --form-font-size: 21px;
+    --form-color: #666666;
+}
 
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.btn
+{
+    text-decoration: none;
+    border-radius: 10px;
+    background-color: var(--color-main);
+    color: white;
+}
+
+.btn-bg {
+    font-size: 30px;
+    padding: 20px 50px;
+}
+
+/*header*/
+
+nav {
+    display: flex;
+    padding: 20px 150px;
+    background-color: #C4C4C41A;
+}
+
+.nav-section {
+    width: 100%;
+}
+
+.nav-section-second {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo {
+    width: 30px;
+}
+
+.nav-link {
+    font-size: 20px;
+    font-weight: 500;
+    color: gray;
+    text-decoration: none;
+}
+
+.nav-link:hover {
+    color: var(--color-main);
+}
+
+.active {
+    color: #52555B;
+}
+
+/*hero*/
+
+.hero {
+    height: 850px;
+    background-image: url('../img/first-background.jpg');
```

Comment:
The background image should be half of the screen. You can review the picture named "homepage-desktop.png".